lib/Transport.js

```diff
@@ -5,6 +5,7 @@ var Transport = module.exports = function(path, auth) {
 };
 
 var request = require("superagent");
+var Perry = require("perry");
```

Comment:
Do we really want to include perry here. We discussed ripping perry out of other repos quite recently.